tests/unit/TestNetRemoteDataStreamingReactors.cxx

```diff
@@ -125,3 +125,9 @@ DataStreamReader::Await(uint32_t* numberOfDataBlocksReceived, DataStreamOperatio
 
     return m_status;
 }
+
+void
+DataStreamReader::Cancel()
+{
+    m_clientContext.TryCancel();
```

Comment:
This is probably worth logging (`LOGD`) as it'd be useful to know when a read is canceled.